packages/brick_supabase/lib/src/query_supabase_transformer.dart

```diff
@@ -259,6 +263,18 @@ class QuerySupabaseTransformer<_Model extends SupabaseModel> {
         return 'adj';
       case Compare.notEqual:
         return 'neq';
+      case Compare.inList:
+        return '';
     }
   }
+
+  String _quoteSupabaseValue(dynamic v) {
+    if (v == null) return 'null';
+    if (v is String) {
+      // Escape double quotes by doubling them, then wrap in double quotes
+      final escaped = v.replaceAll('"', '""');
+      return '"$escaped"';
+    }
+    return v.toString();
```

Comment:
Generally, this can be simplified by 
```dart
  String _quoteSupabaseValue(dynamic v) {
    final asString = v.toString();
    // Escape double quotes by doubling them, then wrap in double quotes
    final escaped = v.replaceAll('"', '""');
    return '"$escaped"';
```

But I don't understand the escaping function - this feels like behavior the implementation should handle because it would unexpectedly mutate the value of the query. Can you describe the use case you're solving for in greater detail?

```suggestion
  String _quoteSupabaseValue(dynamic v) {
    return v.toString();
```